.mypy.ini

```diff
@@ -1,4 +1,4 @@
 [mypy]
-python_version = 3.8
```

Comment:
is this change required?

Reply:
we already had specified python versions >= 3.9 in pyproject.toml and also in our tox tests - so wanted to make sure that we also check for language compatibility from that version onward.